### DISCUSSION TITLE
Cách tương tác với nút trong pygame

### DISCUSSION THREAD

**Discussion by haringuyen01** [codingreshapefuture/Community#20] [Q&A] [answered]
Cách tương tác với nút trong pygame

**Comment by lucthienphong1120** [comment, marked as answer]
Bạn có thể nhận các sự kiện từ pygame và sau đó theo dõi KEYDOWN sự kiện, thay vì nhìn vào các phím được trả về get_pressed()

**Reply by haringuyen01** [reply]
thanks you @lucthienphong1120
